ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -963,7 +965,9 @@ private void updatePath() {
     }
 
     mInnerBottomRightCorner.x = mInnerClipTempRectForBorderRadius.right;
-    mInnerBottomRightCorner.y = mInnerClipTempRectForBorderRadius.bottom * -2;
+    mInnerBottomRightCorner.y = borderWidth.bottom != 0
```

Comment:
Could you explain why don't need this logic for `mInnerTopLeftCorner` and `mInnerTopRightCorner`

Reply:
Yes, it can be added (my bad, I kinda add tunnel vision). In the cases where this bug happened it won't cause any difference though, because top was always 0. Do you want me to make a commit adding the same logic to `mInnerTopLeftCorner` and `mInnerTopRightCorner`?

Reply:
If it doesn't make a difference, we shouldn't add it. So you're saying we don't need similar logic for the top corners?

Reply:
I'm not totally sure. From what I saw from the code, `mInnerClipTempRectForBorderRadius` is set based on the `getBounds` of the drawable. In cases where the bounds don't change, top will always be 0. Is there any possibility of the bounds changing in this case?